### PR TITLE
Added initial support for RPID per service

### PR DESCRIPTION
**Pull request by mmoayyed**
Added initial support for RPID per service

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -181,7 +181,7 @@
     </build>
 
     <properties>
-        <cas.version>4.1.0-SNAPSHOT</cas.version>
+        <cas.version>4.1.0</cas.version>
```
Comment (mmoayyed) [inline pom.xml]: Probably should switch to 4.1.1. The rest is kewl.

**Comment by dima767** [comment]
+1 on 4.1.1 version. Once this is merged, we could cut a release and proceed with it...

File: pom.xml (hunk 1 above)
Comment (scalding) [inline pom.xml]: generally, when writing extension type libraries, you would use the lowest supported version. If there are no API changes between 4.1.0 and 4.1.1 that would affect this module, 4.1.0 would be more appropriate. Has there been?

Comment (mmoayyed) [inline pom.xml]: no per release policy patch releases never have API changes, but there is a chance you may run into dependency issues which is also unlikely but possible.

**Pull request closed without merging**